src/framework/core/js/ModelTransformationTransforms.js

```diff
@@ -739,4 +739,150 @@ var fluid = fluid || fluid_2_0_0;
             (transformSpec.max === undefined ||  transformSpec.max >= value) ? true : false;
     };
 
+    /**
+     *
+     * Convert a string to a Boolean.
+     *
+     * The following are all `false`: `undefined`, `null`, "", "0", "false"
+     *
+     * Everything else is `true`.
+     *
+     * @param value {String} The value to be interpreted.
+     * @returns {Boolean} The interpreted value.
+     */
+    fluid.transforms.stringToBoolean = function (value) {
+        return !Boolean(value === undefined || value === null || value === "" || value === "0" || value === "false");
+    };
+
+    fluid.defaults("fluid.transforms.stringToBoolean", {
+        gradeNames: ["fluid.standardTransformFunction"]
+    });
+
+    /**
+     *
+     * Convert any value into a stringified boolean, i. e. either "true" or "false".  Anything that evaluates to
+     * `true` (1, `true`, "non empty string", {}, et. cetera) returns "true".  Anything else (0, `false`, `null`)
+     * returns "false".
+     *
+     * @param value - The value to be converted to a stringified Boolean.
+     * @returns {string} - A stringified boolean representation of the value.
+     */
+    fluid.transforms.booleanToString = function (value) {
+        return value ? "true": "false";
```

Comment:
Fixed.  I also found someone else's linting errors in the IoC tests and fixed those.